### ISSUE TITLE
empty buff bars, no data update

### ISSUE THREAD

**Issue by johnnyrags** [mkaminsky11/Codex#2]
Hey, running the current pull using the url you have listed in the installation instructions. I configured all classes off except my current class SAM, and the bars will not update when I enter combat and apply a buff or a dot.

I've closed out ACT at least a couple of times now to try and ensure the log lock if reset and everything else.

Thoughts on what may be wrong? or is the plugin link just not current anymore to ACT/Live game?

Thanks

**Comment by mkaminsky11** [comment]
is your overlayplugin up to date? the latest version is 0.16.5.0

![image](https://user-images.githubusercontent.com/4978455/110161316-2f3fd180-7dbb-11eb-80b4-9b035985393f.png)

**Comment by johnnyrags** [comment]
It is v0.16.5.0, yes. 
![act](https://user-images.githubusercontent.com/442175/110162182-9bb2d480-7dab-11eb-8ca1-b40be61c20c7.PNG)

**Comment by johnnyrags** [comment]
I was able to figure this out. Turns out the issue I was running into was the type of miniparse overlay i was selecting. I use MopiMopi for my standard ACT bars, selecting that as an option to build the miniparseoverlay apparently doesn't grab to update the bars.

I created a second one using Ember and it successfully links the buffs to the timer bars. I'm thinking because MopiMopi defaults to a different language and has to be changed to English first, perhaps this is what is breaking the linking? In either case, issue resolved for me, but definitely something that may need more investigation for the instructions side of things.

Thank you for creating this!
![timers](https://user-images.githubusercontent.com/442175/110216369-12170b80-7e6c-11eb-997c-ea2494c6553e.png)
